### PR TITLE
add auto submit option to select action changer

### PR DESCRIPTION
**Pull request by gomayonqui**
Add auto submit option after changing the action in a remote form

### CODE REVIEW

**Comment by edraut** [comment]
Looks great, could you add a jasmine test for this?

**Comment by gomayonqui** [comment]
sure 👍

File: jasmine/spec/hoochSpec.js [hunk 1]
```diff
@@ -122,20 +122,33 @@ describe("hooch", function() {
...
+
+    it('auto-submit', function(){
+      var form = affix('form');
+      var fake_select = form.affix('[data-select-action-changer="search"][data-auto-submit="true"]')
+      var fake_display = fake_select.affix('[data-select-display]')
+      var fake_option1 = fake_select.affix('[data-select-value="pretty/url"][data-select-name="Pretty Stuff"]')
+      var hoo = new hooch.SelectActionChanger(fake_select);
+      var submitCallback = spyOn(hoo, 'submitForm');
+      $('[data-select-value="pretty/url"]').click();
+      expect(submitCallback).toHaveBeenCalled();
```
Comment (edraut) [inline jasmine/spec/hoochSpec.js:149]: 👍

**Comment by edraut** [comment]
nice work! :shipit:

**Pull request merged**